Operations.podspec

```diff
@@ -15,22 +15,33 @@ session Advanced NSOperations: https://developer.apple.com/videos/wwdc/2015/?id=
   s.module_name       = 'Operations'
   s.social_media_url  = 'https://twitter.com/danthorpe'
   s.requires_arc      = true
-  s.platform          = :ios, '8.0'
-  s.default_subspec   = 'Base'
+  s.ios.deployment_target = "8.0"
+  s.osx.deployment_target = "10.10"
 
-  s.subspec 'Base' do |ss|
-    ss.source_files      = 'Operations/**/*.{swift,m,h}'
-    ss.exclude_files     = 'Operations/Extras/**/*.{swift,m,h}'
-  end
```

Comment:
Ah, it seems that without a Base subspec, adding `Operations` results in adding all subspecs as well. Restoring the Base subspec.